### PR TITLE
codewhisperer: update reference setting description

### PR DESCRIPTION
**Pull request by zixlin7**
## Problem
Professional users are not clear that the allow reference setting is controlled by admin
## Solution
Update the description per doc writer

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "AWS.configuration.description.awssam.debug.snippets.api.description": "A new configuration for invoking an AWS Lambda in a CloudFormation template, simulating API Gateway",
     "AWS.configuration.description.awssam.debug.snippets.api.label.cn": "Amazon SAM: API Gateway lambda invoke",
     "AWS.configuration.description.awssam.debug.snippets.api.description.cn": "A new configuration for invoking an Amazon Lambda in a CloudFormation template, simulating API Gateway",
-    "AWS.configuration.description.codewhisperer": "When checked, CodeWhisperer will include suggestions with code references",
+    "AWS.configuration.description.codewhisperer": "When checked, CodeWhisperer will include suggestions with code references. If you authenticate through IAM Identity Center, this setting is controlled by your CodeWhisperer Professional administrator.",
```
Comment (justinmk3) [inline package.nls.json:73]: can we link to the docs? markdown hyperlinks are supported here.

Comment (zixlin7) [reply package.nls.json:73]: This is essentially what the documentation says so I don't see a strong need for linking it here? Seems more straight-forward this way.

**Review by leigaol — APPROVED** [review]

File: package.nls.json (hunk 1 above)
Comment (Will-ShaoHua) [reply package.nls.json:73]: ahh +1 to link to the doc. IMO even though the string is already reflecting the content of our doc, part of CX will still want to know the details if they're really curious the process. By linking our doc, they don't need to reach out to us to resolve their questions.

**Review by Will-ShaoHua — APPROVED** [review]

File: package.nls.json (hunk 1 above)
Comment (zixlin7) [reply package.nls.json:73]: ok I will add the doc link

**Review by andrewyuq — APPROVED** [review]

File: package.nls.json (hunk 1 above)
Comment (zixlin7) [reply package.nls.json]: updated

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**